### PR TITLE
CPLP 2099 Company data overlay changes

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

**Comment by manojava-gk** [comment]
This PR includes these changes

1. Cancel process api method update
2. Min height fix in overlay
3. Registration process info in details overlay

File: cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+) => {
+  if (activeTab === 0) {
+    return t('content.admin.registration-requests.overlay.tab1Title')
+  } else {
+    if (isComplete(applicationChecklist)) {
+      return t('content.admin.registration-requests.overlay.tab2Title').replace(
+        '{status}',
+        t('content.admin.registration-requests.overlay.statusComplete')
+      )
+    } else {
```
Comment (oyo) [inline cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx:54]: else not needed when all previous cases result in a `return`

**Review by oyo — CHANGES_REQUESTED** [review]
some remarks - also change log merge conflict to resolve

File: cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      return t('content.admin.registration-requests.overlay.tab2Title').replace(
+        '{status}',
+        t('content.admin.registration-requests.overlay.statusIncomplete')
+      )
```
[the remark this answers is not in the document]
Comment (manojava-gk) [reply cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx:58]: Thanks Martin, nice concept it is. We should replace our existing approach with this.

**Review by nidhigarg-bmw — DISMISSED** [review]
lgtm

**Review by oyo — CHANGES_REQUESTED** [review]
the conversations are marked as resolved but I can't see the requested changes in the code for example in `CompanyDetailsHelper.tsx`

**Comment by manojava-gk** [comment]
> the conversations are marked as resolved but I can't see the requested changes in the code for example in `CompanyDetailsHelper.tsx`

Something went wrong. I had done all the changes in a new branch and that PR is also merged here(if you remember, I had asked you to approve without review). Not sure what happened in b/w, I'm not able to find that commit at all. Unfortunately I deleted old local repo folder. I cloned again to fix git pull issues. Now I do not have access to my local changes too. I have to rework on this. Give some time to update the PR

File: cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+import {
+  ProgressButtonsProps,
+  ProgressStatus,
+} from 'features/admin/applicationRequestApiSlice'
+import { Trans } from 'react-i18next'
+
+export const isComplete = (applicationChecklist: ProgressButtonsProps[]) =>
+  applicationChecklist.reduce(
+    (a, b) => b && b.statusId === ProgressStatus.DONE,
```
Comment (oyo) [inline cx-portal/src/components/pages/Admin/components/RegistrationRequests/CompanyDetailOverlay/CompanyDetailsHelper.tsx:29]: this must be `(a, b) => a && b.statusId === ....`

**Review by oyo — APPROVED** [review]
lgtm now

**Review by oyo — CHANGES_REQUESTED** [review]
one small change - otherwise looks okay

**Review by oyo — APPROVED** [review]

**Pull request merged**